integrations/github/src/definitions/channels.ts

```diff
@@ -115,7 +115,47 @@ export const channels = {
       text,
     },
   },
+  pullRequestReviewComment: {
```

Comment:
**chore:** I'm not sure I understand the difference between this channel and the one right above `pullRequest`. In my mind, the purpose of the `pullRequest` channel is to allow exchanging comments. If it's not, then we should distinguish both in their respective descriptions.

Reply:
Pull requests can have individual comments that are not part of any review, but they can also have pull request reviews, which are comprised of several comments and these are separate from the main PR thread. For instance, I could write comment `A`, then start a review and in the review write the comment `B`. Then, I finish my review and write the comment `C` outside of the review (in the main thread). If someone afterwards wants to reply to `B`, this comment will be placed in the review thread instead of the main thread to avoid confusion. If all comments were placed in the main PR thread, it could mangle conversations by mixing replies to very old messages with replies to new messages.